pymc3/distributions/multivariate.py

```diff
@@ -505,7 +505,13 @@ def __init__(self, n, p, *args, **kwargs):
     def _random(self, n, p, size=None):
         if size == p.shape:
             size = None
-        return np.random.multinomial(n, p, size=size)
+
+        if p.ndim > 1 and p.shape[0] > 1:
```

Comment:
Yep you are right, it only works for 2D (the last dimension is always the num of category). But if a 1D vector is passed it is also fine:
```python
p=np.asarray([[.25,.25,.25,.25]])
import pymc3 as pm
pm.Multinomial.dist(n=1, p=p).random()
```